### PR TITLE
allow correct input dates according to its constraints while creating Call for Papers

### PR DESCRIPTION
**Pull request by rishabhs95**
using existing date-picker config for `registration-period` to place correct constraints on Call for Paper's date-picker.

Issue #1028.

### CODE REVIEW

File: app/views/admin/cfps/_form.html.haml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 .row
   .col-md-8
     = semantic_form_for(@cfp, :url => admin_conference_program_cfp_path(@conference.short_title),:html => {:multipart => true}) do |f|
-      = f.input :start_date, :as => :string, :input_html => { :id => "conference-start-datepicker", :readonly => "readonly"  }
-      = f.input :end_date, :as => :string, :input_html => { :id => "conference-end-datepicker", :readonly => "readonly"  }
+      = f.input :start_date, as: :string, input_html: { id: 'registration-period-start-datepicker',start_date: @conference.start_date,end_date: @conference.end_date, readonly: 'readonly'  }
+      = f.input :end_date, as: :string, input_html: { id: 'registration-period-end-datepicker', readonly: 'readonly'  }
```
Comment (kalabiyau) [inline app/views/admin/cfps/_form.html.haml]: As you are changing your lines I would recommend to take care of syntax of the other lines as well.

Comment (rishabhs95) [inline app/views/admin/cfps/_form.html.haml]: I have made the syntax uniform as you suggested

**Comment by kalabiyau** [comment]
@rishabhs95 sorry for the confusion, but what I meant is. If you are already touching that file and taking  new has syntax for your lines why not to change the whole file into new colon hash syntax from rocket hash syntax

**Comment by rishabhs95** [comment]
@kalabiyau Okay, I have updated the commit and updated the hash syntax.

**Comment by kalabiyau** [comment]
@rishabhs95 looks good to me

**Comment by hennevogel** [comment]
@kalabiyau Drücken Sie den Merge Knopf! :radio_button:

**Pull request merged**